### ISSUE TITLE
Ehm what's that?

### ISSUE THREAD

**Issue by rugk** [atom-minimap/minimap-symbols#1]
An empty repo?

**Comment by abe33** [comment]
Yup, I made an attempt to implement symbols in the minimap, but the performance was poor and in the meantime the API of the minimap changed and broke it so I never pushed anything. I had in my todo to implement a new kind of decorations in the minimap to enable adding labels above a line (and allowing to display symbols in the minimap), but I still haven't found the time to tackle it.
